### ISSUE TITLE
shell portability problems: dash not supported

### ISSUE THREAD

**Issue by threepistons** [qw3rtman/git-fire#40]
I have version 0.2.3 and I'm using Ubuntu 18.04 with sh symlinked to dash.  When I run git fire against a repo with a stash, the stash isn't branched and pushed.
```
~/git/test(master) » git fire
Switched to a new branch 'fire-master-redacted@example.com-1543921496'
On branch fire-master-redacted@example.com-1543921496
nothing to commit, working tree clean
Host key fingerprint is SHA256:UFAjDQH4wwzmGkeNYpztD5NuaIBVkrdFIuoaEN4vCVQ
+---[ECDSA 256]---+
|o.*E+o**+        |
|+*X+o..o..       |
|=O.B.o.          |
|* +=O  .         |
|o=oo+o  S        |
|o+ o..           |
|o .              |
|                 |
|                 |
+----[SHA256]-----+
X11 forwarding request failed on channel 0
Total 0 (delta 0), reused 0 (delta 0)
To gitlab.example.com:sample/test.git
 * [new branch]      fire-master-redacted@example.com-1543921496 -> fire-master-redacted@example.com-1543921496
Branch 'fire-master-redacted@example.com-1543921496' set up to track remote branch 'fire-master-redacted@example.com-1543921496' from 'origin'.
/usr/local/bin/git-fire: 51: /usr/local/bin/git-fire: [[: not found
```
I can silence the error and push the stash by editing /usr/local/lib/mode_modules/git-fire/git-fire to replace the first line (`#!/usr/bin/env sh`) with either `#!/usr/bin/env bash` or `#!/usr/bin/env zsh`.  I suspect that dash's lack of support for [[ causes this and [it will affect other users of Debian-like distros](https://wiki.ubuntu.com/DashAsBinSh).

**Comment by glensc** [comment]
fixes provided several times, merge capable developer, like @qw3rtman not responding

- https://github.com/qw3rtman/git-fire/pull/39
- https://github.com/qw3rtman/git-fire/pull/34

**Comment by qw3rtman** [comment]
Merged #34, fixing this issue.
